app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/MappingWizardClustersStepActions.js

```diff
@@ -23,3 +27,23 @@ export const fetchTargetClustersAction = url => {
   const uri = new URI(url);
   return _getTargetClustersActionCreator(uri.toString());
 };
+
+const _getQueryHostsActionCreator = (url, hostIDsByClusterID) => dispatch => {
+  const hostIDs = Object.values(hostIDsByClusterID).reduce(
```

Comment:
I'm not sure this is something we actually are too concerned about, but since we do a [node 6 build in travis](https://travis-ci.org/ManageIQ/miq_v2v_ui_plugin/builds/411486035?utm_source=github_status&utm_medium=notification), I just wanted to point out that `Object.values` [is not supported in node 6](https://node.green/#ES2017-features-Object-static-methods-Object-values)

If you had a unit test for this method, that particular build would fail

Reply:
Hmm, okay. I can rewrite with Object.keys. Thanks for the heads up

Reply:
I should add some unit tests too... @michaelkro do you think we shouldn't merge this without a few? don't want to be adding technical debt, but I don't want to hold it up either

Reply:
Unit tests are a welcome addition, but they haven't been a requirement, I think mainly due to the volatility of our current UI.

If you want to write some, go for it, but it's not an issue for merging

Reply:
@michaelkro one more late night comment :)

It looks like we are already using `Object.values` in `searchFilter.js` (part of the PlanWizardVMStep), and I see an `object.values` polyfill in the yarn.lock. I'll fix this in the morning if we need to, but are we sure we can't use it?

Reply:
Looks like you're right.  I just tried running MIQ with node v6.9.1 (the lowest supported version) and things look good.

As I initially stated, I'm not sure if this is a real concern, i.e., I wouldn't have blocked this PR for this.  I just know from prior experience, that, if you try to add a unit test that calls this method, it will fail in the Travis node v6.0.0 run